src/utils.test.js

```diff
@@ -497,4 +499,29 @@ describe('utils', () => {
       expect(logEntry.message).not.toBe(undefined);
     });
   });
+
+  describe('thousandSeparatorNumber function', () => {
+    it('should return correct intl format for numbers in ESP', () => {
+      // Assert
+      expect(thousandSeparatorNumber('es', 100)).toBe('100');
+      expect(thousandSeparatorNumber('es', 1000)).toBe('1.000');
+      expect(thousandSeparatorNumber('es', 10000)).toBe('10.000');
+    });
+
+    it('should return correct intl format for numbers in ENG', () => {
+      // Assert
+      expect(thousandSeparatorNumber('en', 100)).toBe('100');
+      expect(thousandSeparatorNumber('en', 1000)).toBe('1,000');
+      expect(thousandSeparatorNumber('en', 10000)).toBe('10,000');
+    });
+  });
+
+  describe('compactNumber function', () => {
+    it('should return correct compact format for numbers (no matter what language)', () => {
+      // Assert
+      expect(compactNumber(100)).toBe('100');
+      expect(compactNumber(1000)).toBe('1K');
+      expect(compactNumber(10000)).toBe('10K');
+    });
+  });
 });
```

Comment:
Two phases are united: Act & Assert. 
It does not affect the operation at all, it is only a matter of organization. 
I think that @cbernat  has more knowledge about this
You can divide the phases like this:

```
describe('compactNumber function', () => {
    it('should return correct compact format for numbers (no matter what language)', () => {
      // Act
     const value1 = compactNumber(100);
     const value2 = compactNumber(1000);
     const value3 = compactNumber(10000);

      // Assert
      expect(value1).toBe('100');
      expect(value2).toBe('1K');
      expect(value3).toBe('10K');
    });
  });

```

Reply:
Good Idea @jcamposmk!

What is more, you can use [each](https://jestjs.io/docs/api#describeeachtablename-fn-timeout), as [we are already doing here](https://jestjs.io/docs/api#describeeachtablename-fn-timeout):

```js
describe.each`
  rawNumber  | expectedFormatted
  100        | 100
  1000       | 1K
  10000      | 10K
`('compactNumber function', ({ rawNumber, expectedFormatted }) => {
    it(`should return ${expectedFormatted} when the number is ${rawNumber} (no matter what language)`, () => {
      // Act
     const result = compactNumber(rawNumber);

      // Assert
      expect(result).toBe(expectedFormatted);
    });
  });
```